### ISSUE TITLE
Render as Flutter widget rather than PDF?

### ISSUE THREAD

**Issue by jimmyff** [DavBfr/dart_pdf#1511]
As the packages widgets are based on flutter widgets, I wondered if it were possible to output to a flutter widget rather than PDF? I thought this would be a really performant way of previewing the document and then I could render to pdf when required?

Thanks

**Comment by DavBfr** [comment]
No, they look like the Flutter widgets but they are not. It's a implementation of the same API to build PDFs
